<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1556</width>
    <height>954</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>714</width>
      <height>263</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Ressourcen.qrc">:/img/Image/jidan.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Ressourcen.qrc">:/img/Image/xiaoji.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Ressourcen.qrc">:/img/Image/muji.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1556</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Huehnerverwaltung">
    <property name="title">
     <string>&amp;Hühnerverwaltung</string>
    </property>
    <addaction name="actionLegeLe_istung_erfassen"/>
    <addaction name="separator"/>
    <addaction name="action_Stammdaten_2"/>
    <addaction name="separator"/>
    <addaction name="action_Druck"/>
   </widget>
   <widget class="QMenu" name="menu_Anderes">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Anderes</string>
    </property>
    <widget class="QMenu" name="menu_Sprach">
     <property name="title">
      <string>&amp;Sprach</string>
     </property>
     <addaction name="action_Deutsch"/>
     <addaction name="action_Chinesisch"/>
     <addaction name="action_Englisch"/>
    </widget>
    <addaction name="actionFuetterung"/>
    <addaction name="actionZucht"/>
    <addaction name="actionKrankheit"/>
    <addaction name="actionMedikamentengabe"/>
    <addaction name="actionPflege"/>
    <addaction name="actionTierarzt"/>
    <addaction name="actionSonstiges"/>
    <addaction name="menu_Sprach"/>
   </widget>
   <widget class="QMenu" name="menu_Verwaltung">
    <property name="title">
     <string>&amp;Verwaltung</string>
    </property>
    <addaction name="actionNeue_Mitarbeiter"/>
   </widget>
   <addaction name="menu_Huehnerverwaltung"/>
   <addaction name="menu_Anderes"/>
   <addaction name="menu_Verwaltung"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLegeLe_istung_erfassen">
   <property name="text">
    <string>LegeLe&amp;istung erfassen</string>
   </property>
  </action>
  <action name="actionFuetterung">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fütterung</string>
   </property>
  </action>
  <action name="actionZucht">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zucht</string>
   </property>
  </action>
  <action name="actionKrankheit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Krankheit</string>
   </property>
  </action>
  <action name="actionMedikamentengabe">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Medikamentengabe</string>
   </property>
  </action>
  <action name="actionPflege">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pflege</string>
   </property>
  </action>
  <action name="actionTierarzt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Tierarzt</string>
   </property>
  </action>
  <action name="actionSonstiges">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sonstiges</string>
   </property>
  </action>
  <action name="actionNeue_Mitarbeiter">
   <property name="text">
    <string>Neue &amp;Mitarbeiter</string>
   </property>
  </action>
  <action name="action_Deutsch">
   <property name="icon">
    <iconset resource="Ressourcen.qrc">
     <normaloff>:/icons/icon/german9.png</normaloff>:/icons/icon/german9.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Deutsch</string>
   </property>
  </action>
  <action name="action_Chinesisch">
   <property name="icon">
    <iconset resource="Ressourcen.qrc">
     <normaloff>:/icons/icon/china.png</normaloff>:/icons/icon/china.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Chinesisch</string>
   </property>
  </action>
  <action name="action_Englisch">
   <property name="icon">
    <iconset resource="Ressourcen.qrc">
     <normaloff>:/icons/icon/english9.png</normaloff>:/icons/icon/english9.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Englisch</string>
   </property>
  </action>
  <action name="action_Stammdaten_2">
   <property name="text">
    <string>&amp;Stammdaten</string>
   </property>
  </action>
  <action name="action_Druck">
   <property name="text">
    <string>&amp;Druck</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Ressourcen.qrc"/>
 </resources>
 <connections/>
</ui>
